rustls/src/crypto/cipher.rs

```diff
@@ -55,7 +93,11 @@ impl Iv {
     }
 }
 
-pub(crate) fn make_nonce(iv: &Iv, seq: u64) -> [u8; NONCE_LEN] {
+/// Combine an `Iv` and sequence number to produce a unique nonce.
```

Comment:
It feels like maybe `Nonce` should be a newtype wrapper like `Iv`.

Reply:
have added a new commit that does this